client/my-sites/stats/controller.jsx

```diff
@@ -198,14 +196,12 @@ export default {
 				period: activeFilter.period,
 				path: context.pathname,
 			};
-			renderWithReduxStore(
+			context.primary = (
```

Comment:
Instead of:
```js
context.primary = (
   <Element />
);
```

...these could've been turned into:
```js
context.primary = <Element />;
```

Reply:
Did a quick test, seems like this:
```js
// With parentheses
context.primary = (
	<AsyncLoad
		require="my-sites/stats/stats-insights"
		placeholder={ <StatsPagePlaceholder /> }
		{ ...props }
	/>
);

// Without parentheses
context.primary = <AsyncLoad
	require="my-sites/stats/stats-insights"
	placeholder={ <StatsPagePlaceholder /> }
	{ ...props }
/>;

// With parentheses
context.primary = (
	<AsyncLoad />
);
```

Turns into:

```js
// With parentheses
context.primary = (
	<AsyncLoad
		require="my-sites/stats/stats-insights"
		placeholder={ <StatsPagePlaceholder /> }
		{ ...props }
	/>
);

// Without parentheses
context.primary = (
	<AsyncLoad
		require="my-sites/stats/stats-insights"
		placeholder={ <StatsPagePlaceholder /> }
		{ ...props }
	/>
);

// With parentheses
context.primary = <AsyncLoad />;
```

I.e. there's difference how prettier handles elements spanning to multiple lines. 

Fair enough. :-)

Reply:
FYI, I tested also with [calypso-prettier@1.7.4](https://github.com/Automattic/wp-calypso/pull/19519): no difference.